editor/docks/scene_tree_dock.cpp

```diff
@@ -4072,9 +4066,14 @@ void SceneTreeDock::_filter_gui_input(const Ref<InputEvent> &p_event) {
 	}
 
 	if (mb->is_pressed() && mb->get_button_index() == MouseButton::MIDDLE) {
-		filter_quick_menu->clear();
+		if (filter_quick_menu == nullptr) {
+			filter_quick_menu = memnew(PopupMenu);
+			filter_quick_menu->set_theme_type_variation("FlatMenuButton");
```

Comment:
What's the variation for?
EDIT:
Wrong copy paste. It shouldn't have been assigned.